### PR TITLE
Stable/newton

### PR DESCRIPTION
**Pull request by pjbreaux**
@zancas 
#### What's this change do?
Bringing newton up-to-date with mitaka changes.

### CODE REVIEW

**Comment by zancas** [comment]
test this please

**Comment by zancas** [comment]
@pjbreaux can I just say:

**Comment by zancas** [comment]
"test this please" like I did above?

**Comment by pjbreaux** [comment]
retest this please

**Comment by pjbreaux** [comment]
retest this please

**Comment by pjbreaux** [comment]
retest this please

**Comment by pjbreaux** [comment]
retest this please

**Comment by pjbreaux** [comment]
retest this please

**Comment by pjbreaux** [comment]
retest this please

File: systest/scripts/run_neutron_lbaas.sh [hunk 1]
```diff
@@ -30,7 +30,8 @@ sudo -E touch ${NEUTRON_LBAAS_DIR}/neutron_lbaas/tests/tempest/v2/.pytest.rootdi
 # Navigate to the root of the repo, where the tox.ini file is found
 cd ${PROJROOTDIR}
 
-sudo -E bash -c "tox --sitepackages -e tempest -c tox.ini -- \
+sudo -E pip install .
```
Comment (zancas) [inline systest/scripts/run_neutron_lbaas.sh]: Please delete line 33.   This installation is now part of the testrunner.

**Pull request merged**